pkg/kademlia/routing_helpers.go

```diff
@@ -299,18 +299,20 @@ func (rt *RoutingTable) getUnmarshaledNodesFromBucket(bID bucketID) ([]*pb.Node,
 
 // getKBucketRange: helper, returns the left and right endpoints of the range of node ids contained within the bucket
 func (rt *RoutingTable) getKBucketRange(bID bucketID) ([]bucketID, error) {
-	kadIDs, err := rt.kadBucketDB.ReverseList(bID[:], 2)
+	kadBucketIDs, err := rt.kadBucketDB.List(nil, 0)
```

Comment:
Maybe specify the limit, so if it does hit the limit it is visible. I know it's probably never going to happen, but nevertheless.

Reply:
the limit should be bit length (256) + bucket size (20), but I don't think we currently clean up our tree structure sufficiently to enforce that